plugins/catalog-backend/src/ingestion/processors/ldap/config.ts

```diff
@@ -260,6 +260,11 @@ export function readLdapConfig(config: Config): LdapProviderConfig[] {
     };
   }
 
+  function formatFilter(filter?: string): string | undefined {
+    // Squash multiline query filters to a single line query
```

Comment:
Now the comment and changeset may need updating, right

Reply:
I have updated the comment. However, the issue that this change solves is still the same; to be able to configure a long LDAP query over multiple lines in your yaml configuration.